examples/aggregator/module/rust/src/lib.rs

```diff
@@ -106,7 +106,7 @@ impl AggregatorNode {
         //     integrity_tags: vec![],
         // };
         match oak::grpc::client::Client::new_with_label(
-            &oak::node_config::grpc_client("127.0.0.1:8888"),
+            &oak::node_config::grpc_client("https://localhost:8888"),
```

Comment:
@ipetr0v this is the part that I don't understand how it could work before (did it?).

Reply:
I think it didn't. Strange that CI didn't pick it up.